### ISSUE TITLE
OTF fonts cannot be used

### ISSUE THREAD

**Issue by Meorawr** [Stanzilla/WoWUIBugs#164]
**2021-09-15:** This issue has been resolved [on the latest PTR builds for 9.1.5 and 1.14.0](https://github.com/Stanzilla/WoWUIBugs/issues/164#issuecomment-920423713), but still affects the current live version of BCC (2.5.2.40045).

---

OTF fonts are broken as of 2.5.2, 1.14.0, and 9.1.5 and cannot be used with fontstrings in the game client. Support for OTF worked previously in older patches of the game client for each branch, including 2.5.1, 1.13.7, and 9.1.

To test, install an OTF font in the game directory and attempt to use it with a fontstring. A good test case is [Source Sans](https://github.com/adobe-fonts/source-sans/releases/tag/3.046R), for which you can use `SourceSans3-Regular.otf` inside `OTF-source-sans-3.046R.zip`. Place this in the Fonts folder of the game directory and run the following ingame:

```
/run local FS = UIParent:CreateFontString(); FS:SetPoint("CENTER", 0, 10); FS:SetFont("Fonts\\FRIZQT__.ttf", 15); FS:SetText("Test Text");

/run local FS = UIParent:CreateFontString(); FS:SetPoint("CENTER", 0, -10); FS:SetFont("Fonts\\SourceSans3-Regular.otf", 15); FS:SetText("Test Text");
```

The expected result should be two lines of text in the centre of the screen. In the clients where support is broken, you'll only see one line of text.

![image](https://user-images.githubusercontent.com/287102/131667100-dc5f0354-db17-4759-9db6-8bc78701e0c4.png)

On the current 9.1 and 1.13.7 clients, the test case works as expected:

![image](https://user-images.githubusercontent.com/287102/131667070-baed83da-a189-4f02-8521-f29d2869eccf.png)

**Comment by Meorawr** [comment]
For mainline this appears to be resolved in 9.1.5.40196.

![image](https://user-images.githubusercontent.com/287102/133516910-e8c47c3c-3b0e-4a1f-8e50-8d3696ac2604.png)

**Comment by Meorawr** [comment]
Fixed as of 2.5.2.40203 for BCC, which should go live soon.

![image](https://user-images.githubusercontent.com/287102/133695874-5cb6324f-ccb7-472b-be2b-f4546fd18dfe.png)
